rule/json.go

```diff
@@ -105,8 +105,17 @@ func unmarshalExpr(kind string, data []byte) (Expr, error) {
 		var or exprOr
 		e = &or
 		err = or.UnmarshalJSON(data)
+	case "percentile":
+		var percentile exprPercentile
+		e = &percentile
+		err = percentile.UnmarshalJSON(data)
+	case "FNV":
```

Comment:
Maybe it is better to have all operators in lowercase here? To avoid any mistakes.

Reply:
:heavy_check_mark: